Homework_11/src/testng/BicycleTest.java

```diff
@@ -0,0 +1,54 @@
+package test.java;
+
+import org.testng.annotations.*;
+
+import tat.maven.Bicycle;
+
+import static org.testng.Assert.*;
+
+/**
+ * Created by Nick on 30/10/16.
+ */
+public class BicycleTest {
+    Bicycle bicycle;
+    double start;
+
+    @DataProvider(name = "points")
+    public Object[][] getPoints() {
+        return new Object[][]{
+                {2, 0, 2},
```

Comment:
Line 19 and 20 is the same equivalence class, you have to set positive, negative and zero values in different combinations